server/routes/index.ts

```diff
@@ -27,6 +27,10 @@ export default function routes(services: Services): Router {
   const downloadNomisConfigRoutes = makeDownloadNomisConfigRoutes()
   get('/nomis-report-config/incident-types.csv', downloadNomisConfigRoutes.incidentTypes)
   get('/nomis-report-config/incident-type/:type/questions.csv', downloadNomisConfigRoutes.incidentTypeQuestions)
+  get(
+    '/nomis-report-config/incident-type/:type/prisoner-roles.csv',
```

Comment:
But prisoner roles are not specific of any Report type tho no?

Reply:
it appears that they _are_. there's a global set of roles that exist, but what's available per-question is given here.
<img alt="incident type config" src="https://github.com/user-attachments/assets/eac5e51b-ffe1-43f6-b872-e98d73f55269">

Reply:
Ah OK, I didn't realise this 👌

Reply:
me neither